apps/backend/fixtures/lambda/batch.json

```diff
@@ -1,8 +1,8 @@
 {
   "period": {
-    "from": "",
+    "from": "2023-06-01",
     "to": ""
   },
   "program": "SBC",
-  "generateReport": true
+  "reportEnabled": false
```

Comment:
I'm confused about the rename here?

Reply:
I see it's also renamed in reconcile.json and in batch-reconcile.ts, was this just for naming clarity?

Reply:
Yes. generateReport sounded more the action of actually generating it, so I have used this for the func name that is called if "reportEnabled" is true.